libs/flowbite-angular/src/lib/components/sidebar/sidebar-menu.theme.ts

```diff
@@ -31,7 +31,7 @@ export const sidebarMenuTheme: SidebarMenuTheme = createTheme({
       disabled: 'hidden',
     },
     color: {
-      primary: 'border-primary-200 dark:border-primary-700',
+      primary: 'border-gray-200 dark:border-gray-800',
```

Comment:
Keep primery with its values, and then call either dark, or get a new color like grayas you have done, add the key in the navbarcolors, for every component using it's type, and then call in the documentation the color dark/gray